Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionCodeGenerator.cs

```diff
@@ -86,7 +92,7 @@ public static string GenerateWrapperCode(InputActionAsset asset, Options options
             }
 
             // Begin class.
-            writer.WriteLine($"public class @{options.className} : IInputActionCollection, IDisposable");
+            writer.WriteLine($"public partial class @{options.className} : IInputActionCollection, IDisposable");
```

Comment:
Could we use templates for generated code, like how MonoBehaviours and other Unity assets work?

Reply:
Maybe. For now, seems a bit overkill and probably makes for an additional upgrade headache.